hw/ip/otbn/doc/_index.md

```diff
@@ -606,6 +606,9 @@ The following actions are taken when OTBN detects a recoverable error:
 
 The host software can start another operation on OTBN after a recoverable error was detected.
 
+When {{< regref "CTRL.all_errs_fatal" >}} is set recoverable errors become fatal errors.
+In this case the actions above are ignored in favour of [actions for fatal errors](#design-details-fatal-errors).
```

Comment:
Nit: This would be easier to read with a comma after "case".

That said, I wonder whether it would be clearer to say something like "In this case, the \[actions for a fatal error\](#design-details-fatal-errors) are run instead of the ones above." ?

Reply:
Simplified this to a single sentence. It doesn't explicitly say that the recoverable error actions aren't triggered now but I feel it's obvious enough from the context.